base_layer/key_manager/src/lib.rs

```diff
@@ -11,3 +11,29 @@ pub mod mnemonic_wordlists;
 #[allow(clippy::unused_unit)]
 #[cfg(feature = "wasm")]
 pub mod wasm;
+
+use tari_common::hashing_domain::*;
```

Comment:
I'd prefer something more like this:

```suggestion

hash_domain!(KeyManagerHashDomain, "com.tari.keymanager");

pub fn create_hasher<D:Digest>(label: &'static str) -> DomainSeparatedHasher<....> {
   DomainSeparatedHasher::new::<D>(label)
}




```

Reply:
or using the `create_hasher!()` macro directly

Reply:
or finally, one method for each use case...
```suggestion
hash_domain!(KeyManagerHashDomain, "com.tari.keymanager");

pub fn create_passphrase_hasher() -> DomainSeparatedHasher<VarBlake, KeyManagerHashDomain> {}

pub fn create_cipher_seed_mac_hasher() -> Domain....
```